app/services/holding_requests_adapter.rb

```diff
@@ -0,0 +1,172 @@
+  end
+
+  # Determine whether or not the electronic resources are accessible using umlaut
+  # @return [TrueClass, FalseClass]
+  def umlaut_accessible?
+    getit_accessible? || doc_electronic_access.empty? || !@document.umlaut_fulltext_eligible?
+  end
+
+  # Retrieve only the ELF holding records
+  # @return [Hash] ELF holding information
+  def doc_holdings_elf
+    doc_holdings.select { |_id, h| h['location_code'].start_with?('elf') }
+  end
+
+  # Retrieve only the records for physical holdings
+  # @return [Hash] physical holding information
+  def doc_holdings_physical
+    doc_holdings.reject { |_id, h| h['location_code'].start_with?('elf') }
+  end
+
+  # Retrieve the physical holdings records sorted by location code
+  # @return [Hash] sorted physical holding information
+  def sorted_physical_holdings
+    doc_holdings_physical.sort_by do |_id, h|
+      @bib_data_service.holding_locations.keys.index(h['location_code'])
+    end
+  end
+
+  # Determine whether or not the catalog record is for a periodical
+  # @return [TrueClass, FalseClass]
+  def journal?
+    @document.fetch('format', []).include?('Journal')
+  end
+
+  # Retrieve the publication date for the catalog record
+  # @return [String] the date value
+  def pub_date
+    @document.key?('pub_date_start_sort') ? @document['pub_date_start_sort'] : 0
+  end
+
+  # Methods for holding values
+  # Should these be refactored into static methods
+  # (or should a decorator be used for holding values?)
+
+  # Generate a full string for the location values
+  # @param location [Hash] location information
+  # @return [String] the location string
+  def location_full_display(location)
+    return location['library']['label'] if location.fetch('label') == ''
+    location['library']['label'] + ' - ' + location['label']
+  end
+
+  # Retrieve the location rules from holding values
+  # @param holding [Hash] the holding values
+  # @return [Hash] location values
+  def holding_location_rules(holding)
+    loc_code = holding.fetch('location_code', nil)
+    return loc_code if loc_code.nil?
+    @bib_data_service.holding_locations[loc_code.to_sym]
+  end
+
+  # Generate the label for a location from the holding values
+  # @param holding [Hash] the holding values
+  # @return [String] the location label
+  def holding_location_label(holding)
+    location = holding_location_rules(holding)
+    location.nil? ? holding['location'] : location_full_display(location)
+  end
+
+  # Retrieve the call number from holding values
+  # @param holding [Hash] the holding values
+  # @return [String] the call number
+  def call_number(holding)
+    holding['call_number_browse'] || holding['call_number']
+  end
+
+  # Determine whether or not the holding is for a repository item
+  # @return [TrueClass, FalseClass]
+  def repository_holding?(holding)
+    holding['dspace']
+  end
+
+  # Determine whether or not the holding is for a ReCAP item
```

Comment:
Technically this would only test for "scsb" partner recap holdings. PUL has a lot of "recap" items we own at location code that don't start with "scsb".  We might want to use a different name for this method.

Reply:
@kevinreiss Thank you correcting this, I have updated the method names for `HoldingRequestsAdapter#recap_holding?` and `PhysicalHoldingsMarkupBuilder.recap_location?` in order to have this resolved.

However there appear to be cases where this is in place outside of this PR (please see https://github.com/pulibrary/orangelight/blob/master/app/helpers/orangelight/browsables_helper.rb#L3).  I can address these within the scope of this PR as well, or open another issue for this.

Reply:
@jrgriffiniii Looking at this a little more I can see that all the scsb related code that was added to the helpers uses a prefix of ```recap_``` for the method names when I really should have used something more specific like```scsb_```.  These all seem to only provide behaviors specific to the SCSB partner records. https://github.com/pulibrary/orangelight/pull/1100/files?diff=unified#diff-a213c754c47e851ef2a146f5397d902eR192 is another example of this. For the sake of clarity do you think it is worth going through all of them and making this change?